app/src/main/res/layout/feedinfo.xml

```diff
@@ -159,7 +159,38 @@
                 android:enabled="false"
                 android:textColor="?android:attr/textColorPrimary"
                 tools:background="@android:color/holo_red_light"
-                android:checked="false"/>
+                android:checked="false" />
+
+            <android.support.v7.widget.GridLayout
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_marginTop="8dp"
+                app:columnCount="2"
+                app:rowCount="1">
+
+                <TextView
+                    android:id="@+id/txtvFeedAutoDelete"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:text="@string/auto_delete_label"
+                    app:layout_row="0"
+                    app:layout_column="0"
+                    app:layout_gravity="center_vertical"
+                    android:layout_marginRight="10dp" />
+
+                <Spinner
+                    android:layout_width="wrap_content"
+                    android:layout_height="match_parent"
+                    android:id="@+id/spnAutoDelete"
+                    android:entries="@array/spnAutoDeleteItems"
+                    android:layout_marginTop="8dp"
+                    app:layout_row="0"
+                    app:layout_column="1"
+                    android:spinnerMode="dropdown"
+                    app:layout_gravity="center"
+                    android:dropDownWidth="wrap_content"
+                    android:clickable="false" />
```

Comment:
Why is clickable set to false?
When testing this PR on virtual devices, the drop down menu would not open, unless I set clickable to true?